src/adapters/Cornerstone/MeasurementReport.js

```diff
@@ -319,9 +317,8 @@ export default class MeasurementReport {
                   );
 
             if (toolClass) {
-                const measurement = toolClass.getMeasurementData(
-                    measurementGroup
-                );
+                const measurement =
```

Comment:
Seems like a lot of formatting changes - if you want to run the formatter, do it as a separate PR that is just that change so that the review only has the substantive changes.